.github/ISSUE_TEMPLATE/3-gh-bug-feature-report.yml

```diff
@@ -1,7 +1,7 @@
-name: Bug Report - MSSQL + GitHub Copilot
+name: Bug Report / Feature Request - MSSQL + GitHub Copilot
 description: File a bug report / feature request to help us improve GitHub Copilot integration with the MSSQL extension.
 title: "[GitHub Copilot - Bug] / Feature request: "
-labels: ["bug", "PrPr", "Area - GitHub Copilot", "Triage: Needed"]
+labels: ["Bug", "Enhancement", "PrPr", "Area - GitHub Copilot", "Triage: Needed"]
```

Comment:
Rather than automatically applying both labels, we should probably have neither and let the triager add the correct one?